helper/utils.go

```diff
@@ -0,0 +1,18 @@
+package helper
+
+func Flatten(s [][]byte) []byte {
```

Comment:
```go
func Flatten(collection [][]byte) []byte {
	totalLen := 0
	for i := range collection {
		totalLen += len(collection[i])
	}

	result := make([]T, 0, totalLen)
	for i := range collection {
		result = append(result, collection[i]...)
	}

	return result
}
```

Maybe we can rearrange Flatten operation like this? Or We can do some benchmark

Reply:
This method is no longer needed, so it will be deleted.